files/scripts/syncd.sh

```diff
@@ -52,6 +52,24 @@ function wait_for_database_service()
     done
 }
 
+function getBootType()
+{
+    case "$(cat /proc/cmdline | grep -o 'SONIC_BOOT_TYPE=\S*' | cut -d'=' -f2)" in
```

Comment:
>SONIC_BOOT_TYPE= [](start = 41, length = 16)

We should handle both cases for backward-compatible with 201803:

fast-reboot
SONIC_BOOT_TYPE=fast-reboot
Otherwise we cannot fast-reboot from 201803 into 201811.